.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,56 @@
+name: gh-pages
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  skip:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Skip job"
+
+  before:
+    runs-on: ubuntu-latest
+    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - run: echo "not contains '[skip ci]'"
+
+  docs:
+    runs-on: ubuntu-latest
+    needs: before
+    env:
+      nim_version: '1.2.0'
```

Comment:
Can this be changed to 'devel --latest'?

Reply:
Sounds good to me, too.

Reply:
Thx. Fixed.

Reply:
I changed NimVersion from 1.2.0 to `devel --latest` .
And I added `continue-on-error: true`.